### DISCUSSION TITLE
Xyce pametric plot

### DISCUSSION THREAD

**Discussion by dwarning** [ra3xdh/qucs_s#692] [Q&A]
The problem is similar like #161 but seems not solved yet.
I try to make an CV plot with Xyce:

![xyce_plot](https://github.com/ra3xdh/qucs_s/assets/18441932/a68ec658-9d0c-448d-b2e7-4244edd73422)

The resulting data file (attached) looks very well and is plotable with gnuplot.
[nmos_cgs_vgs_xyce_script.txt](https://github.com/ra3xdh/qucs_s/files/14974626/nmos_cgs_vgs_xyce_script.txt)

How I can get the curve into Qucs-s?

**Comment by ra3xdh** [comment]
This is as the same as #161 You have a set of 1x1 point plots, but not as a smooth plot. I am aware about this problem and there is no workaround without extension of XML dataset format. Qucs plotting engine cannot understand that a set of 1x1 plots is a smooth curve. You may set *Style: Stars* in the diagram properties dialog and you will see the data points.

**Comment by ra3xdh** [comment]
Closing as duplicate of #161

**Comment by ra3xdh** [comment]
I have converted this to discussion, because it is duplicate issue. You have to find a way to assemble a set of 1x1 point plots into one curve as it was done for Ngspice #277 I am not sure if it is possible for Xyce.

**Comment by dwarning** [comment]
Do you looked into the attached result file? The 4th data set {-IMG(I(VG))/(2*PI*1E6)} FREQ Number> is dependent, the 3rd dataset also dependent and the PlotVs() emulation is able to choose a second dependent vector as the independent for plotting. But here it is not working.

**Comment by ra3xdh** [comment]
It is the same issue as #161 PlotVs may works wrong on parameter sweep data. There is a subtask in #161 to fix this. Don't expect a quick solution.
